tinygrad/runtime/graph/hcq.py

```diff
@@ -54,7 +54,11 @@ def __init__(self, jit_cache: List[ExecItem], input_rawbuffers: List[Buffer], va
 
     for j,ji in enumerate(jit_cache):
       enqueue_dev = ji.prg.dev if (is_exec_prg:=isinstance(ji.prg, CompiledRunner)) else Device[ji.bufs[1].device] #type:ignore
```

Comment:
Kinda hacky, but could save one line by removing the following assert and declaring `enqueue_dev: HCQCompiled`.